scripts/dev/generate_epJSON_schema/idd_parser.py

```diff
@@ -152,7 +152,7 @@ class Data:
 
 
 def parse_idd(data):
-    root = {'$schema': "http://json-schema.org/draft-04/schema#", 'properties': {}}
+    root = {'$schema': "https://json-schema.org/draft-07/schema#", 'properties': {}}
```

Comment:
What are the implications of changing to draft 7?  Does it unlock more capabilities in the schema?

Reply:
Yes, it does. There was only 1 minor change needed to update our schema to draft 7 (exclusiveMinimum and exclusiveMaximum are now values instead of boolean, basically you put either minimum or exclusiveMinimum, not both, now).